## if..else condition
#### if statement is a programming conditional statement that, if proved true, performs a function or displays information.
+ Conditional operators for if 
+ if..elif..else 
+ Ternary operator 

In [5]:
# Even odd programme
n = input("Enter any number")
n = int(n)
if n%2 == 0:
    print("Your entered number is even")
else:
    print("You entered a odd number")

Enter any number 45


You entered a odd number


In [6]:
# Even odd programme
n = input("Enter any number")
n = int(n)
if n%2 == 0:
    print("Your entered number is even")
else:
    print("You entered a odd number")

Enter any number 400


Your entered number is even


In [7]:
# Even odd programme
n = input("Enter any number")
n = int(n)
if n%2 == 0:
    print("Your entered number is even")
else:
    print("You entered a odd number")

Enter any number 415.35


ValueError: invalid literal for int() with base 10: '415.35'

+ ValueError: invalid literal for int() with base 10: '415.35'

### Conditional operators for if 

In [17]:
print(4 == 4)
print(6 != 7)
print(9 == -9)
print(9 < 8)
print(8 > 9)
print((4+5) == 9 )
print(3 > 2 and 9 > 5)
print(3 > 4 and 9 > 3)
print(3 > 4 or 9 > 3)
print(not 4 == 4)
print(not 4 != 4 )
print(not 4 > 10)

True
True
False
False
False
True
True
False
True
False
True
True


In [22]:
positive_int =['1','2','3','4','5','6','7']
negative_int =["-1","-2","-3","-4","-5","-6","-7"]
complex_num  =["1j","2j","3j","4j","5j","6j","7j"]
number = input("Enter a number from given lists")
if number in positive_int:
    print(f"{number} is psotive_integer")
elif number in negative_int:
    print(f"{number} is negative_integer")
elif number in complex_num:
    print(f"{number} is complex number")
else:
    print("Enter a Valid number from lists")

Enter a number from given lists 5


5 is psotive_integer


In [23]:
positive_int =['1','2','3','4','5','6','7']
negative_int =["-1","-2","-3","-4","-5","-6","-7"]
complex_num  =["1j","2j","3j","4j","5j","6j","7j"]
number = input("Enter a number from given lists")
if number in positive_int:
    print(f"{number} is psotive_integer")
elif number in negative_int:
    print(f"{number} is negative_integer")
elif number in complex_num:
    print(f"{number} is complex number")
else:
    print("Enter a Valid number from lists")

Enter a number from given lists 45


Enter a Valid number from lists


In [24]:
positive_int =['1','2','3','4','5','6','7']
negative_int =["-1","-2","-3","-4","-5","-6","-7"]
complex_num  =["1j","2j","3j","4j","5j","6j","7j"]
number = input("Enter a number from given lists")
if number in positive_int:
    print(f"{number} is psotive_integer")
elif number in negative_int:
    print(f"{number} is negative_integer")
elif number in complex_num:
    print(f"{number} is complex number")
else:
    print("Enter a Valid number from lists")

Enter a number from given lists 1j


1j is complex number


In [21]:
positive_int =['1','2','3','4','5','6','7']
negative_int =["-1","-2","-3","-4","-5","-6","-7"]
complex_num  =["1j","2j","3j","4j","5j","6j","7j"]
number = input("Enter a number from given lists")
if number in positive_int:
    print(f"{number} is psotive_integer")
elif number in negative_int:
    print(f"{number} is negative_integer")
elif number in complex_num:
    print(f"{number} is complex number")
else:
    print("Enter a Valid number from list")

Enter a number from given lists -7


-7 is negative_integer


## Ternary operator 

In [25]:
n = input("Enter any number")
n = int(n)
message ="Number is even" if n%2==0 else "Number is odd"
print(message)

Enter any number 14


Number is even
